src/vscode-dts/vscode.proposed.chatTab.d.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+	/**
+	 * The tab represents an interactive window.
+	 */
+	export class TabInputChat {
+		/**
+		 * The uri of the history notebook in the interactive window.
+		 */
+		readonly providerId: string;
```

Comment:
providerId as a general concept might go away, do want to use this for something?

Reply:
It would be nice for an extension that is providing a chat interface to be able to know what chat inputs belong to it vs other chat providers

Reply:
Also oops that comment says "uri of the history notebook". My copy and paste is exposed

Reply:
In the agent world this is a bit different, no one extension owns the chat interface.